src/ListBox/README.md

```diff
@@ -0,0 +1,132 @@
+# ListBox
+
+Provides a ListBox component that can takes Option components as children and handles selection of and navigation through the Options.
+
+## API
+
+### `children` (Option component array; required)
```

Comment:
Is there a better way to describe this prop so that users don't pass in an explicit `children` prop?

Reply:
Okay, thank you.